src/errors.rs

```diff
@@ -40,6 +41,10 @@ pub enum Error {
     InvalidEntryActions(BTreeMap<Vec<u8>, EntryError>),
     /// Key does not exist
     NoSuchKey,
+    /// Key already exists
+    KeysExists(Entries),
```

Comment:
Should be `KeysExist`.  Also, the doc comment could probably be expanded a bit, e.g.
```rust
/// The key(s) of the entry or entries contained in this error already exist
```